new_data_contribution.md

```diff
@@ -0,0 +1,272 @@
+
+When you are ready to archive your lab data, please submit the following request in the [Arcus Help Center](https://pm.arcus.chop.edu/servicedesk/customer/portal/6/create/256) to begin the data contribution process.
+
+During the orientation for your Arcus Lab, you should have received an orientation to the Project Template structure. A review of this information is available in Module 2 [ADD LINK LATER] and in this [Arcus forum post](https://forum.arcus.chop.edu/t/the-arcus-project-template/255). Prior to archiving, the research data and all associated reference information needs to be organized in the project template directory preloaded into your lab. If you need assistance in arranging your data, please request a consultation by the Arcus Library Sciences team using the [following link](https://jira.arcus.chop.edu:8443/servicedesk/customer/portal/6/create/278?q=rdm&q_time=1690816635148).
+
+## Why contribute?
+
+One of the primary functions of Arcus is to link otherwise disparate data sources to enable new opportunities for CHOP researchers and to accelerate the path to discovery. Contributing data to Arcus allows us to achieve this goal in that research data sets can be discovered along with and linked to other research data sets, clinical data sources, and computational tools.
+
+We have designed the data contribution process to be iterative and to complement the diverse needs of the research institute’s user community. Some Arcus projects that are hosted on or dependent on other Arcus or DBHi services (registry or cohort creation, for example) also have a data contribution component that is part and parcel of this research effort.
+
+### Data Reuse
+
+**Collate data in Arcus for use in lab**
+
+As mentioned, contributing research data is prerequisite to working with that data in an Arcus lab. Once data is stored in Arcus it can be both integrated with a variety of data sources and successfully delivered to a lab. Our privacy team has regulatory language that allows for expansive and evolving integration. Arcus labs are available at a persistent web URL that facilitates collaborative research across multiple users.
+
+**Open Science initiatives**
+
+Broadly speaking, Open Science Initiatives promote making publicly funded research data more widely available and collecting and organizing these data in a way that makes them reusable. For example, as of 2023, all NIH funded studies regardless of size must develop a [data management and sharing plan](https://www.research.chop.edu/sites/default/files/2022-11/DBHi_Converting_Resource_DMSP.pdf) which stipulates data is ultimately deposited in a searchable data repository. Archiving your data in Arcus ensures that your data is in compliance with this requirement.
+
+### Grant Requirements
+
+**FAIR Data**
+
+![FAIR data principles graphic](media/FAIR_data_principles.png)
```

Comment:
```suggestion
![FAIR data principles graphic](media/FAIR_data_principles.png)<!-- style = "max-width: 600px" -->
```
I think it looks better slightly smaller but maybe that's just me?  Take it or leave it!